basic_pitch/models.py

```diff
@@ -204,17 +202,6 @@ def model(
         )(x)
 
     # contour layers - fully convolutional
-    x_contours = tfkl.Conv2D(
-        n_filters_contour,
-        (5, 5),
-        padding="same",
-        kernel_initializer=_initializer(),
-        kernel_constraint=_kernel_constraint(),
-    )(x)
-
-    x_contours = tfkl.BatchNormalization()(x_contours)
-    x_contours = tfkl.ReLU()(x_contours)
-
```

Comment:
Please note I'm including this cleanup that was discussed in #23, but since we create a layer and overwrite it, I think that these lines are just meaningless and have no effect on the output.

If you'd like to leave this implementation as it is, I would like to re-add this code as it is and leave references for this bug as a comment in the code.